test/pco-license/CFAReclaimerFacet.ts

```diff
@@ -0,0 +1,275 @@
+import { expect, use } from "chai";
+import chaiAsPromised from "chai-as-promised";
+import { ethers, getNamedAccounts, network } from "hardhat";
+import { BigNumber } from "ethers";
+import { solidity } from "ethereum-waffle";
+import { smock } from "@defi-wonderland/smock";
+import { addDays, getUnixTime, startOfToday } from "date-fns";
+import BaseFixtures from "./CFABasePCO.fixture";
+import { rateToPurchasePrice } from "../shared";
+
+use(solidity);
+use(chaiAsPromised);
+use(smock.matchers);
+
+describe("CFAReclaimerFaceit", async function () {
+  describe("claim", async () => {
+      it("emits the licenseReclaimed event", async () => {
+        const { basePCOFacet, mockParamsStore, paymentToken, ethersjsSf } = await BaseFixtures.afterPayerDelete();
+        const { bidder } = await getNamedAccounts();
+
+        const contributionRate = BigNumber.from(100);
+        const forSalePrice = await rateToPurchasePrice(
+          mockParamsStore,
+          contributionRate
+        );
+        const requiredBuffer = await ethersjsSf.cfaV1.contract
+        .connect(await ethers.getSigner(bidder))
+        .getDepositRequiredForFlowRate(
+          paymentToken.address,
+          contributionRate
+        );
+        const totalCollateral = forSalePrice.add(requiredBuffer);
+
+        // Transfer payment token for buffer
```

Comment:
This should be done via a `transferFrom` inside the contract, right?

Reply:
This transfer the required amount to the bidder address that can later on call the reclaim function with enough balanc of the payment token so I don't think it should

Reply:
Ok I think I see. The bidder doesn't have enough ETHx? It should start with 10 ETHx from the setup [here](https://github.com/Geo-Web-Project/core-contracts/blob/main/test/shared.ts#L78) which should be enough.

Reply:
Oh didn't knew, changed it right now